openmetadata-ui/src/main/resources/ui/src/components/Settings/CustomProperty/EditCustomPropertyModal/EditCustomPropertyModal.tsx

```diff
@@ -100,6 +100,8 @@ const EditCustomPropertyModal: FC<EditCustomPropertyModalProps> = ({
         const updatedValues = uniq([...value, ...(enumConfig?.values ?? [])]);
         form.setFieldsValue({ customPropertyConfig: updatedValues });
       },
+      open: false,
```

Comment:
@chirag-madlani , Any reason we are passing this prop as false?

Reply:
Not needed as options are same as what user has added in select